### ISSUE TITLE
Docker --security-opt fails on mac 

### ISSUE THREAD

**Issue by clearstorm-tech** [meta-llama/llama-stack#166]
Docker desktop v4.11.1 for Mac Sequoia

      llama stack build --template local-ollama --name stack-test-docker --image-type docker

Configuring API `inference`...
=== Configuring provider `remote::ollama` for API inference...
Enter value for host (default: localhost) (required):
Enter value for port (required): 4001
> Please enter the supported model your provider has for inference: Llama3.1-8B-
Instruct

Configuring API `memory`...
=== Configuring provider `meta-reference` for API memory...
> Please enter the supported memory bank type your provider has for memory: vect
or

Configuring API `safety`...
=== Configuring provider `meta-reference` for API safety...
Do you want to configure llama_guard_shield? (y/n): n
Do you want to configure prompt_guard_shield? (y/n): n

Configuring API `agents`...
=== Configuring provider `meta-reference` for API agents...
Enter `type` for persistence_store (options: redis, sqlite, postgres) (default: sqlite): redis

Configuring RedisKVStoreConfig:
Enter value for namespace (optional): llama-stack
Enter value for host (default: localhost) (required):
Enter value for port (default: 6379) (required):

Configuring API `telemetry`...
=== Configuring provider `meta-reference` for API telemetry...

> YAML configuration has been written to `/Users/<username>/Documents/tools/llama-stack/tmp/configs/stack-test-docker-run.yaml`.
You can now run `llama stack run stack-test-docker --port PORT`
Dockerfile created successfully in /var/folders/7f/xb01fy750wgczjwyw164pch00000gq/T/tmp.ZYi5Waxvvk/DockerfileFROM python:3.10-slim
WORKDIR /app

RUN apt-get update && apt-get install -y        iputils-ping net-tools iproute2 dnsutils telnet        curl wget telnet        procps psmisc lsof        traceroute        bubblewrap        && rm -rf /var/lib/apt/lists/*

RUN pip install llama-stack
RUN pip install pandas codeshield aiosqlite httpx psycopg2-binary fastapi scipy sentencepiece blobfile numpy redis scikit-learn pillow ollama chardet pypdf matplotlib transformers uvicorn faiss-cpu nltk tqdm fire
RUN pip install torch --index-url https://download.pytorch.org/whl/cpu
RUN pip install sentence-transformers --no-deps
ENTRYPOINT ["python", "-m", "llama_stack.distribution.server.server"]

ADD tmp/configs/stack-test-docker-build.yaml ./llamastack-build.yaml
ADD tmp/configs/stack-test-docker-run.yaml ./llamastack-run.yaml

     + docker build --security-opt label=disable -t llamastack-stack-test-docker -f /var/folders/7f/xb01fy750wgczjwyw164pch00000gq/T/tmp.ZYi5Waxvvk/Dockerfile /Users/<username>/Documents/tools/llama-stack
Error response from daemon: The daemon on this platform does not support setting security options on build

Failed with error for security opts

     Failed to build target stack-test-docker with return code 1

Running without the --security-opt flag works

     docker build -t llamastack-stack-test-docker -f /var/folders/7f/xb01fy750wgczjwyw164pch00000gq/T/tmp.QMw22Q87GF/Dockerfile /Users/<username>/Documents/tools/llama-stack

**Comment by russellb** [comment]
should be fixed in #167
